<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="APT04-1DKF7G3" failures="1" tests="2" name="Personal loan" time="0.038" errors="0" timestamp="2024-03-17T15:13:41 IST">
  <testcase classname="FrameworkDay1.TestNG" name="Demo" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at FrameworkDay1.TestNG.Demo(TestNG.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="FrameworkDay1.TestNG" name="Demosecondtest" time="0.001"/>
  <testcase classname="FrameworkDay1.Day2" name="Ploan" time="0.0">
    <ignored/>
  </testcase> <!-- Ploan -->
</testsuite> <!-- Personal loan -->
